GrindingCity.WebApi/DTOs/Requests.cs

```diff
@@ -0,0 +1,6 @@
+﻿using GrindingCity.WebApi.Models;
+
+namespace GrindingCity.WebApi.DTOs;
+
+public record CreateBuildingRequest(decimal price, BuildingType buildingType);
+public record CreateResourseRequest(string title, decimal price);

```

Comment:
typo in Resource

Reply:
If you look into Lowered code parameters in constructors of `records` **Are properties** 
![image](https://user-images.githubusercontent.com/48443516/231598440-b4ebf09d-6543-4365-82eb-dbd451cf61ed.png)

there for they should start with capital letter